### PR TITLE
Add bash completion for DNS settings in services

### PR DESCRIPTION
**Pull request by albers**
Ref: #27567
please schedule for 1.13.0
ping @sdurrheimer for zsh completion

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐮

**Comment by AkihiroSuda** [comment]
LGTM

File: contrib/completion/bash/docker [hunk 1]
```diff
@@ -2643,6 +2643,9 @@ _docker_service_update() {
 	if [ "$subcommand" = "create" ] ; then
 		options_with_args="$options_with_args
 			--container-label
+			--dns
+			--dns-option
```
Comment (sdurrheimer) [inline contrib/completion/bash/docker]: missing a `s`: `--dns-options` (https://github.com/docker/docker/pull/27567/files#diff-728a7fe135201867a6b4b645adc9553dR480)

Comment (albers) [inline contrib/completion/bash/docker]: @sdurrheimer Seems to be correct, though:
```bash
$ docker service create --dns-option
flag needs an argument: --dns-option
See 'docker service create --help'.
$ docker service create --dns-options
unknown flag: --dns-options
See 'docker service create --help'.
```
Maybe this was changed in the meantime, see also #28369

Comment (sdurrheimer) [inline contrib/completion/bash/docker]: OH! didn't see this one, sorry.

**Pull request merged**